### PR TITLE
[graphics] Added the ability to click on maps from the base

### PR DESCRIPTION
**Pull request by GauchKirill**
- Added the ability to activate actions

### CODE REVIEW

File: src/graphics/command.cpp [hunk 1]
```diff
@@ -12,10 +12,23 @@ void Command::set_card(PlayerCard *card) {
...
+        minion->press();
+    }
+    check_commands();
+}
+
+void Command::set_shown_card(PlayerCard *card) {
+    if (!card)
+        return;
+    if (Action *action = dynamic_cast<Action *>(card); action) {
+        events.push_back(std::make_pair(Type::action_on_base, action));
```
Comment (aleksplast) [inline src/graphics/command.cpp:28]: In current state, action cant be played on base, what is this for?

Comment (GauchKirill) [reply src/graphics/command.cpp:28]: For the future fitch
:)

Comment (aleksplast) [reply src/graphics/command.cpp:28]: Okay

**Comment by aleksplast** [comment]
LGTM

**Pull request merged**